lib/iris/plot.py

```diff
@@ -662,7 +663,18 @@ def _ensure_cartopy_axes_and_determine_kwargs(x_coord, y_coord, kwargs):
     # Ensure the current axes are a cartopy.mpl.GeoAxes instance.
     axes = kwargs.get('axes')
     if axes is None:
-        _replace_axes_with_cartopy_axes(cartopy_proj)
+        if (isinstance(cs, iris.coord_systems.RotatedGeogCS) and
```

Comment:
> It is less of an issue in our user community as the use of GeogCS for limited area models is unusual

True, one of my motivations is that having a specific caveat in the code can make it difficult to understand the context at a later date.

> @cpelley Please may you clarify: are you explicitly asking for this behaviour to be provided for and tested for GeogCS as part of this change?

Not at all.  I don't want to get in the way of the clear benefit this provides as it is.

Would be good to think about the iris-cartopy relationship with regarding to coordinate systems in future.  Doesn't feel right.  That's for another time.